packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/ToggleSwitch.tsx

```diff
@@ -39,12 +46,24 @@ export interface ToggleSwitchProps extends Props {
 	 */
 	labelPosition?: LabelPosition;
 	/**
-	 * Whether the toggle has a tooltip.
+	 * Displays a border above the toggle switch label and toggle.
+	 */
+	labelBorder?: boolean;
+	/**
+	 * If JS is not available the toggle is not clickable and shows a tooltip.
 	 * The default is false.
 	 */
-	tooltip?: boolean;
```

Comment:
how come this is being removed? couldn't find a mention of it in the pr description

Reply:
The `tooltip` prop was also not used anywhere previously, its a bizarre bit of implementation.

Reply:
reinstated this prop so no longer relevant